linode/helper/framework_provider_model.go

```diff
@@ -5,6 +5,26 @@ import (
 	"github.com/linode/linodego"
 )
 
+func GetFrameworkProviderModelFromProviderConfig(config *Config) *FrameworkProviderModel {
```

Comment:
```suggestion
func GetFrameworkProviderModelFromSDKv2ProviderConfig(config *Config) *FrameworkProviderModel {
```
Maybe more clear to state the config is from SDKv2 provider?